.github/workflows/ci.yaml

```diff
@@ -121,6 +121,7 @@ jobs:
           - { name: "uml", backend: "--fakemachine-backend=uml" }
         test:
           - { name: "recipes", case: "recipes" }
+          - { name: "arch", case: "arch" }
```

Comment:
if you just want arch tests to run in a fakemachine and not in the `--disable-fakemachine` case, you can do this instead:

```
include:
  - backend: { name: "arch", backend: "--fakemachine-backend=uml" }
    test: { name: "arch", case: "arch" }
```